playbooks/build-ansible-collection/run.yaml

```diff
@@ -15,7 +15,7 @@
     - name: Generate version number for ansible collection
       args:
         chdir: "~/{{ item.src_dir }}"
-      shell: "if test -f 'galaxy.yml'; then ~/.local/bin/generate-ansible-collection; fi"
+      shell: "if test -f 'galaxy.yml'; then ~/.local/bin/generate-ansible-collection --for-release; fi"
```

Comment:
build-ansible-collection is also run for 'normal' CI jobs, not just for the 'release' job, I think the --for-release flag should only be run if we're running because someone triggered a the job by pushing a tag

Reply:
I guess I'd need to copy this directory to a new one `build-ansible-collection-for-release` and do the change there. Or is there a cleaner way to do this @goneri?